README.md

```diff
@@ -124,6 +119,12 @@ that will be completed:
 
 This is how to run the playbook:
 ```shell
+make dependencies
```

Comment:
If `make dependencies` is a formal dependency in Makefile, this line can be removed.